### PR TITLE
Update getting started page

### PR DESCRIPTION
**Pull request by dipinknair**
- [x] Add grids
- [x] Update the version for embedded in linux
- [x] Move Installation guide to a separate page

Relates to #560 

**Current version**

image

**New version**

image

### CODE REVIEW

**Review by koubaa — APPROVED** [review]

**Comment by RobPasMue** [comment]
I'd use a 3 column grid if possible rather than 2-1-2-1

File: doc/source/getting_started/index.rst [hunk 1]
```diff
@@ -1,150 +1,74 @@
+.. _ref_getting_started:
+
 ===============
-Getting started
+Getting Started
```
Comment (RobPasMue) [inline doc/source/getting_started/index.rst:4]:
```suggestion
Getting started
```

Adhere to Google doc style

**Review by RobPasMue — CHANGES_REQUESTED** [review]

**Comment by dipinknair** [comment]
New updates  @RobPasMue @koubaa 

- cards arranged in 2 columns
- Updated docs as per @sbradrick feedback.
- Fixed broken links
- added link check to make file. can be used with ``doc/make.bat linkcheck`` to see broken or redirected links in docs
- removed project index from main page
- renamed docs section titles

**Review by koubaa — APPROVED** [review]

File: doc/source/examples/index.rst [hunk 2]
```diff
@@ -12,14 +12,12 @@ an embedded instance or remote session.
    .. grid-item-card:: Embedding Examples
       :link: https://embedding.examples.mechanical.docs.pyansys.com/index.html
       :text-align: center
-      :margin: 2 2 0 0
```
Comment (Revathyvenugopal162) [inline doc/source/examples/index.rst:15]: if you are removing the margin, i would recommend to add gutter for the cards. we follow 2 2 3 4 pattern as i wrote in some of the comments below.

File: doc/source/getting_started/index.rst [hunk 3]
```diff
@@ -1,150 +1,72 @@
...
-a wheelhouse archive on an isolated system with a fresh Python installation or on a
-virtual environment.
+This section provides the following information.
 
-For example, on Linux with Python 3.9, unzip the wheelhouse archive and install it with
-this code:
 
-.. code::
+.. grid:: 2
+    :gutter: 2
```
Comment (Revathyvenugopal162) [inline doc/source/getting_started/index.rst:10]:
```suggestion
    :gutter: 2 2 3 4
```

File: doc/source/getting_started/index.rst [hunk 4]
```diff
@@ -1,150 +1,72 @@
...
-If you want to install PyMechanical on a computer without access to the internet,
-you can download a wheelhouse archive that corresponds to your
-machine architecture from the `Releases page <https://github.com/ansys/pymechanical/releases>`_ 
-of the PyMechanical repository.
-
-Each wheelhouse archive contains all the Python wheels necessary to install
-PyMechanical from scratch on Windows and Linux for Python 3.9 through Python 3.12. You can install
-a wheelhouse archive on an isolated system with a fresh Python installation or on a
-virtual environment.
+This section provides the following information.
```
Comment (Revathyvenugopal162) [inline doc/source/getting_started/index.rst:6]:
```suggestion
```

File: doc/source/getting_started/index.rst [hunk 5]
```diff
@@ -1,150 +1,72 @@
...
+        :link: versioning
+        :link-type: doc
+        :text-align: center
 
-The way that you verify your installation depends on whether you want to run
-Mechanical using a remote session or an embedded instance.
-Before running either, you must first verify that you can find
-the installed version of Mechanical using the ``ansys.tools.path`` package.
-This package is required to use PyMechanical.
+        Versioning requirements and support for Ansys Mechanical
```
Comment (Revathyvenugopal162) [inline doc/source/getting_started/index.rst:24]:
```suggestion
        Versioning requirements and support for Ansys Mechanical.
```

File: doc/source/getting_started/index.rst [hunk 6]
```diff
@@ -1,150 +1,72 @@
...
 
-.. code:: pycon
 
-   >>> from ansys.tools.path import find_mechanical
-   >>> find_mechanical()
+.. grid:: 2
+    :gutter: 2
```
Comment (Revathyvenugopal162) [inline doc/source/getting_started/index.rst:28]:
```suggestion
    :gutter: 2 2 3 4
```

File: doc/source/getting_started/index.rst [hunk 7]
```diff
@@ -1,150 +1,72 @@
...
 
-   or
+    .. grid-item-card:: :fas:`fa-solid fa-gears`  Launching PyMechanical
```
Comment (Revathyvenugopal162) [inline doc/source/getting_started/index.rst:30]:
```suggestion
    .. grid-item-card:: :fas:`fa-solid fa-gears` Launching PyMechanical
```

File: doc/source/getting_started/index.rst [hunk 8]
```diff
@@ -1,150 +1,72 @@
...
+        :text-align: center
 
-If you are on Windows with Python 3.9, unzip the ``ansys-mechanical-core-v0.11.dev0-wheelhouse-Windows-3.9``
-wheelhouse archive to a ``wheelhouse`` directory and then install it using ``pip`` like
-in the preceding example.
+        How to install and verify PyMechanical
 
-Verify your installation
-------------------------
+    .. grid-item-card:: :fas:`fa-solid fa-code-compare`  Versioning
```
Comment (Revathyvenugopal162) [inline doc/source/getting_started/index.rst:19]:
```suggestion
    .. grid-item-card:: :fas:`fa-solid fa-code-compare` Versioning
```

File: doc/source/getting_started/index.rst [hunk 9]
```diff
@@ -1,150 +1,72 @@
...
+    :gutter: 2
 
-   or
+    .. grid-item-card:: :fas:`fa-solid fa-gears`  Launching PyMechanical
+        :link: running_mechanical
+        :link-type: doc
+        :text-align: center
 
-   >>> find_mechanical(version=231)  # for specific version
+        Steps to run PyMechanical
```
Comment (Revathyvenugopal162) [inline doc/source/getting_started/index.rst:35]:
```suggestion
        Steps to run PyMechanical.
```

File: doc/source/getting_started/index.rst [hunk 10]
```diff
@@ -1,150 +1,72 @@
...
+        Steps for creating mechanical docker images
 
-.. code:: pycon
 
-   >>> from ansys.tools.path import save_mechanical_path, find_mechanical
-   >>> save_mechanical_path("home/username/ansys_inc/v231/aisol/.workbench")
-   >>> path = get_mechanical_path()
-   >>> print(path)
+.. grid:: 2
+    :gutter: 2
```
Comment (Revathyvenugopal162) [inline doc/source/getting_started/index.rst:46]:
```suggestion
    :gutter: 2 2 3 4
```

File: doc/source/getting_started/index.rst [hunk 11]
```diff
@@ -1,150 +1,72 @@
...
+    .. grid-item-card:: :fab:`fa-brands fa-docker`  Docker setup  
+        :link: docker
+        :link-type: doc
+        :text-align: center
 
-If you install Ansys in a directory other than the default or typical location,
-you can save this directory path using the ``save_mechanical_path`` function. Then use
-``get_mechanical_path`` and ``version_from_path`` functions to verify the path and version.
-For more details, refer to the :ref:`ref_ansys_tools_path_api`.
+        Steps for creating mechanical docker images
```
Comment (Revathyvenugopal162) [inline doc/source/getting_started/index.rst:42]:
```suggestion
        Steps for creating mechanical docker images.
```

File: doc/source/getting_started/index.rst [hunk 12]
```diff
@@ -1,150 +1,72 @@
...
+        :link: running_mechanical
+        :link-type: doc
+        :text-align: center
 
-   >>> find_mechanical(version=231)  # for specific version
+        Steps to run PyMechanical
 
-   ('C:/Program Files/ANSYS Inc/v231/aisol/bin/winx64/AnsysWBU.exe', 23.1)  # windows
-   ('/usr/ansys_inc/v231/aisol/.workbench', 23.1) # Linux
+    .. grid-item-card:: :fab:`fa-brands fa-docker`  Docker setup
```
Comment (Revathyvenugopal162) [inline doc/source/getting_started/index.rst:37]:
```suggestion
    .. grid-item-card:: :fab:`fa-brands fa-docker` Docker setup  
```

File: doc/source/getting_started/index.rst [hunk 13]
```diff
@@ -1,150 +1,72 @@
...
-   pip install ansys-mechanical-core -f wheelhouse --no-index --upgrade --ignore-installed
+    .. grid-item-card:: :fas:`fa-regular fa-screwdriver-wrench`  Installation guide
+        :link: installation
+        :link-type: doc
+        :text-align: center
 
-If you are on Windows with Python 3.9, unzip the ``ansys-mechanical-core-v0.11.dev0-wheelhouse-Windows-3.9``
-wheelhouse archive to a ``wheelhouse`` directory and then install it using ``pip`` like
-in the preceding example.
+        How to install and verify PyMechanical
```
Comment (Revathyvenugopal162) [inline doc/source/getting_started/index.rst:17]:
```suggestion
        How to install and verify PyMechanical.
```

File: doc/source/getting_started/installation.rst [hunk 14]
```diff
@@ -0,0 +1,142 @@
+.. _ref_installation:
+==================
```
Comment (Revathyvenugopal162) [inline doc/source/getting_started/installation.rst:2]:
```suggestion

```

File: doc/source/getting_started/installation.rst [hunk 15]
```diff
@@ -0,0 +1,142 @@
...
+
+To run PyMechanical, you must have a licensed copy of Ansys Mechanical
+installed locally. The version installed dictates the interface and
+features that are available to you.
+
+PyMechanical is compatible with Mechanical 2023 R1 and later on Windows
+and Linux. For more information please refer to  :ref:`ref_versioning` documentation.
+Later releases provide significantly better support and features.
+
```
Comment (Revathyvenugopal162) [inline doc/source/getting_started/installation.rst:13]:
```suggestion
```

File: doc/source/getting_started/installation.rst [hunk 16]
```diff
@@ -0,0 +1,142 @@
...
+`venv -- Creation of virtual environments <https://docs.python.org/3/library/venv.html>`_.
+
+Install the latest package from `PyPi
+<https://pypi.org/project/ansys-mechanical-core/>`_ with this command:
+
+.. code::
+
+   pip install ansys-mechanical-core
+
+
```
Comment (Revathyvenugopal162) [inline doc/source/getting_started/installation.rst:31]:
```suggestion
```

File: doc/source/getting_started/installation.rst [hunk 17]
```diff
@@ -0,0 +1,142 @@
...
+For example, on Linux with Python 3.9, unzip the wheelhouse archive and install it with
+this code:
+
+.. code::
+
+   unzip ansys-mechanical-core-v0.11.dev0-wheelhouse-Linux-3.9 wheelhouse
+   pip install ansys-mechanical-core -f wheelhouse --no-index --upgrade --ignore-installed
+
+If you are on Windows with Python 3.9, unzip the ``ansys-mechanical-core-v0.11.dev0-wheelhouse-Windows-3.9``
+wheelhouse archive to a ``wheelhouse`` directory and then install it using ``pip`` like
```
Comment (Revathyvenugopal162) [inline doc/source/getting_started/installation.rst:53]:
```suggestion
wheelhouse archive to a ``wheelhouse`` directory and then install it using ``pip`` as
```

File: doc/source/getting_started/installation.rst [hunk 18]
```diff
@@ -0,0 +1,142 @@
...
+in the preceding example.
+
+Verify your installation
+------------------------
+
```
Comment (Revathyvenugopal162) [inline doc/source/getting_started/installation.rst:58]:
```suggestion
```

File: doc/source/getting_started/installation.rst [hunk 19]
```diff
@@ -0,0 +1,142 @@
...
+    Product Version:231
+    Software build date:Wednesday, August 10, 2022 4:28:15 PM
+
+If you see a response from the server, you can begin using Mechanical
+as a service. For information on the PyMechanical interface, see
+:ref:`ref_mechanical_user_guide`.
+
+Verify an embedded instance
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
```
Comment (Revathyvenugopal162) [inline doc/source/getting_started/installation.rst:115]:
```suggestion
```

File: doc/source/getting_started/installation.rst [hunk 20]
```diff
@@ -0,0 +1,142 @@
...
+   available to install using pip:
+
+   ``pip install ansys-mechanical-env`` 
+
+To use the script, prepend it to any invocation of Python:
+
+.. code::
+
+    mechanical-env python
+
```
Comment (Revathyvenugopal162) [inline doc/source/getting_started/installation.rst:130]:
```suggestion
```

File: doc/source/getting_started/installation.rst [hunk 21]
```diff
@@ -0,0 +1,142 @@
...
+Verify an embedded instance
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Verify your installation by loading an embedded instance of Mechanical in Python.
+
+.. note::
+   If you are running on Linux, you must set some environment variables for
+   embedding of Mechanical in Python to work. A script that sets these variables is
+   available to install using pip:
+
```
Comment (Revathyvenugopal162) [inline doc/source/getting_started/installation.rst:122]:
```suggestion
```

File: doc/source/getting_started/running_mechanical.rst [hunk 22]
```diff
@@ -1,8 +1,8 @@
 .. _using_standard_install:
 
-***********************************************
-Using PyMechanical from a standard installation
-***********************************************
+======================
```
Comment (Revathyvenugopal162) [inline doc/source/getting_started/running_mechanical.rst:3]:
```suggestion
```

File: doc/source/getting_started/versioning.rst [hunk 23]
```diff
@@ -1,6 +1,8 @@
-**********
+.. _ref_versioning:
+
+==========
```
Comment (Revathyvenugopal162) [inline doc/source/getting_started/versioning.rst:3]:
```suggestion
```

File: doc/source/getting_started/wsl.rst [hunk 24]
```diff
@@ -1,8 +1,8 @@
 .. _ref_guide_wsl:
 
-PyAnsys libraries in the Windows Subsystem for Linux and Docker
-###############################################################
-
+===========================
```
Comment (Revathyvenugopal162) [inline doc/source/getting_started/wsl.rst:3]:
```suggestion
```

File: doc/source/links.rst [hunk 25]
```diff
@@ -1,5 +1,5 @@
 .. #Pyansys
-.. _pyansys: https://docs.pyansys.com/dev/
+.. _pyansys: https://docs.pyansys.com/version/dev/
```
Comment (Revathyvenugopal162) [inline doc/source/links.rst:2]:
```suggestion
.. _pyansys: https://docs.pyansys.com/
```
default landing page is the dev version.

**Review by Revathyvenugopal162 — APPROVED** [review]
looks good some minor comments.

File: doc/source/examples/index.rst (hunk 2 above)
Comment (dipinknair) [reply doc/source/examples/index.rst]: Reverting back to margin

File: doc/source/api/path.rst [hunk 26]
```diff
@@ -2,6 +2,7 @@
 
 Ansys path tools
 ================
+
 These methods on the ansys-tools-path module provide helper functions for configuration and discovery
```
Comment (RobPasMue) [inline doc/source/api/path.rst:6]:
```suggestion
These methods on the ``ansys-tools-path`` module provide helper functions for configuration and discovery
```

**Review by RobPasMue — APPROVED** [review]
LGTM! Great overall docs edit @dipinknair

**Pull request merged**